darshan-util/pydarshan/darshan/cli/summary.py

```diff
@@ -436,15 +436,21 @@ def register_figures(self):
             opcounts_mods.append("H5D")
         elif "H5F" in self.report.modules:
             opcounts_mods.append("H5F")
+        elif "PNETCDF_VAR" in self.report.modules:
```

Comment:
@tylerjereddy Shouldn't this be `if` rather than `elif`, otherwise we'd run into issues if a log had both PnetCDF and HDF5 modules active?

Reply:
Probably, and can't the same be said for the `elif` above as well with the new config modularity? I suspect we need more test files to catch that stuff.

Reply:
I fixed s.t. a different `if`/`elif` clause is used for `PNETCDF_FILE`/`PNETCDF_VAR` than for `H5F`/`H5D`. Otherwise, it doesn't look like opcounts plots would be generated for both PNETCDF and HDF5 if a module happened to have both (probably very rare).

I'm not sure if there's other places in this logic that need fixed. My understanding from the preceding comment is that you only want one of `H5F`/`H5D` or `PNETCDF_FILE`/`PNETCDF_VAR`, as the `H5D` and `PNETCDF_VAR` modules know how to additionally include the file-level data.

I spot checked a couple of summary reports and confirmed tests succeed with the changes, so think I'm content with the changes for now, but could still be some corner cases to sort out.